### PR TITLE
Add asset checks and project cleanup

### PR DESCRIPTION
**Pull request by izzye84**
Organized assets, jobs, schedules, and sensors into their respective directories.  

Also added a few asset checks as a new feature.

### CODE REVIEW

File: hooli_data_eng/resources/__init__.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    if os.getenv("DAGSTER_CLOUD_DEPLOYMENT_NAME", "") == "data-eng-prod":
+        return "PROD"
+    return "LOCAL"
+
+
+# The dbt file dbt_project/config/profiles.yaml
+# specifies what databases to targets, and locally will
+# execute against a DuckDB
+
+DBT_PROJECT_DIR = file_relative_path(__file__, "../dbt_project")
```
Comment (slopp) [inline hooli_data_eng/resources/__init__.py:46]: Are these relative paths still correct? I'd expect they need to be tweaked, though I'm not 100% sure

**Review by slopp — APPROVED** [review]
This looks mostly good to me, two things:

1. I think we need to update the file relative paths for dbt stuff? Maybe?
2. We mayneed to update the hooli_data_eng_tests to use the new location of things as well?

File: hooli_data_eng/resources/__init__.py (hunk 1 above)
Comment (slopp) [reply hooli_data_eng/resources/__init__.py:46]: I believe that is what is causing the branch deployment failure above

Comment (izzye84) [reply hooli_data_eng/resources/__init__.py]: I just fixed the paths, but haven't taken a look at the tests.  I'll get to those in a bit.

**Pull request merged**